.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 install: npm install
-script: npm test
+script: NYPL_CORE_VERSION=v1.x-rom-com-prerelease-1.1 npm test
```

Comment:
There's a risk that application code will depend on this hard-coded value and allow tests to pass and then cause a failure in the deployed app (where the above var is not set). Instead of setting the above var for all tests, can you temporarily set it around the specific test that needs it? e.g.
```
  describe('deliveryLocationsByCustomerCode', () => {
    beforeEach(() => {
      process.env.NYPL_CORE_VERSION = 'v1.x-rom-com-prerelease-1.1'
    })
    afterEach(() => {
      process.env.NYPL_CORE_VERSION = undefined
    })
    ...
```

Reply:
that sounds much better. thanks